include/net/nrf_cloud.h

```diff
@@ -74,6 +74,9 @@ extern "C" {
 /** Default valid duration for JWTs generated by @ref nrf_cloud_jwt_generate */
 #define NRF_CLOUD_JWT_VALID_TIME_S_DEF		(10 * 60)
 
+/** Value used to disable sending a timestamp. */
+#define NRF_CLOUD_NO_TIMESTAMP -1
```

Comment:
Since the timestamp is supposed to be in the Unix epoch, I suppose `0` would be an alternative.

Reply:
But note that I just moved this definition from line 534 and added a comment.

Reply:
I've used 0 elsewhere for a timeout. For a time it's a bit more questionable because 0 is a valid time, although not really possible.
@glarsennordic Originally, you pointed out the use of `-1` for indefinite timeout. What would you use here given this is time, not timeout?

Reply:
@trantanen

If it were a timeout, I would agree `-1` might be a questionable choice, but since this is a timestamp (and refers to a point in time rather than to an amount of time), I don't think the `-1 = SYS_FOREVER_MS ` convention applies.

I am neutral between `0` and `-1` in this case. Either can technically be interpreted as a point in Unix time, but either is an equally unlikely point in time for a microcontroller to want to reference.

Reply:
I am learning towards 0 because timestamps are stored in an `int64_t` variable, and so a negative number could happen (in 292 million years...).  0 means 0:00 UTC on Jan 1 1970, which will of course never happen again.

Reply:
I'm fine either way (`-1` or `0`) so feel free to choose what suites you better.